### DISCUSSION TITLE
对改进弹出式菜单的几点建议

### DISCUSSION THREAD

**Discussion by cloudherder** [shuax/MouseInc.Settings#220] [Ideas]
感谢作者写了这么棒的软件！MouseInc是我离不开的软件之一，省了很多键盘的敲击。在使用的过程中，觉得弹出式菜单还可以改进一点，具体来说，是界面做成象FireFox的插件Context Search那样：
![QQ图片20220113091244](https://user-images.githubusercontent.com/12610495/149248394-42e0d71c-0ea3-45b6-b286-1136980f962c.png)
     主要是两点：
     一是增加一个文本输入框，用来自动贴入粘贴板的内容，或者接收键盘输入，作为命令的参数。
     二是用格子按钮展示菜单项，即使没有图标和层级显示，只要可以设置醒目的大字符标签和背景色也足够了。这在信息展示上更密集，在视觉识别上比菜单更快捷一些。
     这样改进之后，MouseInc可以替代的功能会更多。比如，作为启动应用的快捷面板，直接调用命令行什么的。

**Comment by shuax** [comment]
我不太会做界面。

**Comment by cloudherder** [comment]
呃，这就没办法了

**Reply by 6Tom** [reply]
某种情况下，弹出菜单比图形界面是有优点的。

那就是你可以自行设置屁股后面那个字母快捷键**(&R)**，背熟了可以直接全键盘盲打，比图形界面省事太多。
